Makefile

```diff
@@ -17,6 +17,9 @@ LAST_API_URL := $(shell if [ -f .build-artefacts/last-api-url ]; then cat .build
 MAPPROXY_URL ?= //wmts{s}.geo.admin.ch
 MAPPROXY_TECH_URL ?= //wmts{s}.
 LAST_MAPPROXY_URL := $(shell if [ -f .build-artefacts/last-mapproxy-url ]; then cat .build-artefacts/last-mapproxy-url 2> /dev/null; else echo '-none-'; fi)
+VECTORTILES_URL ?= //vectortiles{s}.geo.admin.ch
+VECTORTILES_TECH_URL ?= //vectortiles{s}.
+LAST_VECTORTILES_URL := $(shell if [ -f .build-artefacts/last-vectortiles-url ]; then cat .build-artefacts/last-vectortiles-url 2> /dev/null; else echo '-none-'; fi)
```

Comment:
We have urls for development of vetcortiles : https://vectortiles.dev.bgdi.ch/ch.swisstopo.vegetation.3d/20170126_wsl/tileset.json